examples/regression/new_run_regression.py

```diff
@@ -305,12 +305,11 @@ def main():
         model.set_qiskit_processor(processor_simulation)
         valid_test(dataflow, q_device, "test", model, device, qiskit=True)
 
+        # final valid
+        valid_test(dataflow, q_device, "valid", model, device, True)
```

Comment:
Note: originally, this would raise an error since outside of the try/except, qiskit hasn’t been imported. I made the choice to move it inside the try so we don’t have to worry about the qiskit dependency if a user just wants to run it with pure TQ.